ci/ansible/roles/pulp-build/templates/pulpcore.json.jj2

```diff
@@ -0,0 +1,90 @@
+{
+  "builders":
+  [
+    {
+      "type": "virtualbox-ovf",
+      "source_path": "{{ ovf_path }}",
+      "ssh_username": "vagrant",
+      "ssh_password": "vagrant",
+      "ssh_port": 22,
+      "ssh_wait_timeout": "60s" ,
+      "headless": true,
+      "boot_wait": "10s",
+      {% raw %}
+      "vboxmanage": [
+        ["modifyvm", "{{.Name}}", "--memory", "4096"]
+      ],
+      {% endraw %}
+      "shutdown_command": "echo 'vagrant' | sudo -S shutdown -P now"
+    },
+    {
+      "type": "qemu",
+      "accelerator": "kvm",
+      "iso_url": "{{ img_path }}",
+      "iso_checksum_type": "none",
+      "qemu_binary": "qemu-kvm",
+      "ssh_username": "vagrant",
+      "ssh_password": "vagrant",
+      "ssh_port": 22,
+      "ssh_wait_timeout": "60s",
+      "headless": true,
+      "boot_wait": "10s",
+      "disk_image": true,
+      "disk_size": 40960,
+      "qemuargs": [
+        [ "-m", "4096M" ]
+      ],
+      "shutdown_command": "echo 'vagrant' | sudo -S shutdown -P now"
+    }
+  ],
+  "provisioners":
+  [
+    {
+      "type":"shell",
+      "only": ["virtualbox-ovf"],
+      "inline": [
+        "sudo mkdir -p /tmp/vboxguest",
+        "sudo mount -t iso9660 -o loop /home/vagrant/VBoxGuestAdditions.iso /tmp/vboxguest",
+        "cd /tmp/vboxguest",
+        "sudo dnf -y install dkms kernel-devel-$(uname -r) gcc",
+        "sudo ./VBoxLinuxAdditions.run",
+        "cd /tmp",
+        "sudo umount /tmp/vboxguest",
+        "sudo rmdir /tmp/vboxguest",
+        "rm /home/vagrant/VBoxGuestAdditions.iso"
+      ]
+    },
+    {
+      "type":"ansible",
+      "playbook_file": "deploy-pulp3.yml",
+      "user": "vagrant"
+    },
+    {
+      "type":"shell",
+      "inline": [
+        "sudo rm /etc/yum.repos.d/epel.repo",
+        "sudo dnf -y install git httpie",
+        "echo 'machine localhost' >> /home/vagrant/.netrc",
+        "echo 'login admin' >> /home/vagrant/.netrc",
+        "echo 'password admin' >> /home/vagrant/.netrc",
+        "sudo sed -i 's/enforcing/permissive/' /etc/selinux/config",
+        "echo 'Welcome to the Pulp3 dev environment!' >> sudo /etc/motd" 
+      ]
+    }
+  ],
+  "post-processors": 
+  [
+    [
+      {
+        "type": "vagrant",
+        "vagrantfile_template": "Vagrantfile"
+      },
+      {
+        "type": "vagrant-cloud",
+        "access_token": "0v2o8xCp8Ynnfg.atlasv1.RMc1BFNzAsRiyhMN5mgByb2CtSVkzo1AYZM28SWmJaTqZ2yjf0GG2aGkftqwh1ySEPs",
```

Comment:
please remove this token from here. it should be stored as a secret in jenkins. you should also revoke it and generate a new one.

Reply:
Created a new token in Jenkins and modified the scripts to use this token for uploading boxes.